### PR TITLE
Add Charts Page using the Metrics API

### PR DESCRIPTION
**Pull request by dguenther**
Adds a Charts link to the header, and a Charts page that displays charts from the Metrics API.

### CODE REVIEW

File: src/components/Charts/GeneralChart.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      />
+      <Axis orientation='left' numTicks={8} tickTransform='translate(-8, 0)' hideAxisLine />
+      <Axis
+        orientation='bottom'
+        numTicks={5}
+        tickLabelProps={() => ({ fill: '#000' })}
+        tickTransform='translate(0, 18)'
+        hideAxisLine
+      />
+      <Tooltip<Metric>
```
Comment (brekk) [inline src/components/Charts/GeneralChart.tsx:67]: Is this a generic?

File: src/components/Charts/GeneralChart.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        showVerticalCrosshair
+        snapTooltipToDatumX
+        renderTooltip={({ tooltipData, colorScale }) => {
+          if (
+            colorScale === undefined ||
+            tooltipData === undefined ||
+            tooltipData.nearestDatum === undefined
```
Comment (brekk) [inline src/components/Charts/GeneralChart.tsx:74]: Minor but kinda verbose.

**Review by brekk — APPROVED** [review]
Minor comments but pretty dope esp. for an offsite project.

File: src/components/Charts/GeneralChart.tsx (hunk 1 above)
Comment (dguenther) [reply src/components/Charts/GeneralChart.tsx:67]: Yep, this requires a generic to type the tooltipData in the `renderTooltip` callback.

File: src/components/Charts/GeneralChart.tsx (hunk 2 above)
Comment (dguenther) [reply src/components/Charts/GeneralChart.tsx]: 👍 Shortened this to one condition, good call

**Pull request merged**